### PR TITLE
2019 midyear update

### PR DESCRIPTION
**Pull request by plaindocs**
Not much here yet.

### CODE REVIEW

**Comment by ericholscher** [comment]
Alright, I pushed up a bunch of copy, but now we need to have landing pages for the salary survey and the east coast request for venues, so we can link to them :/

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+Hey there documentarians.
+The newsletter team is taking the month off,
+but that doesn't mean the community is standing still.
+We decided to send out a community update to let you know what's going on with the community. 
+
+Salary Survey
+-------------
+
+The biggest thing that we're excited about is a new :doc:`Salary Survey </surveys/salary-survey-sep-2019>`.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
The thing that we're most excited about is our new :doc:`Salary Survey </surveys/salary-survey-sep-2019>`.
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+Our goal is to help people understand the pay scales and benefits in the software industry around documentation.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
Our goal is to get a clear idea of the pay scales and benefits in the software industry around documentation.
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+This will help people get a better understanding of what an appropriate salary is,
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
Publishing the results will help people get a better understanding of what an appropriate salary is,
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+and then negotiate for a better one.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
and negotiate a better one.
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+The survey is 21 questions broken up into 4 parts.
+Please fill it out here: https://www.surveymonkey.com/r/wtd-salary-2019
+
+The salary will be open until October 15th, 2019,
+after that we will process the data and publish the results.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
after that we will process the data and publish the anonymized results.
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+but many different ways.
+
+.. _Mohit Patel:  https://mohitpatel.design/2019/08/13/write-the-docs-internship-retrospective/
+.. _13 interviews with people in the community: http://www.writethedocs.org/hiring-guide/#community-spotlight
+
+A US East Conference
+--------------------
+
+We're also excited to start planning another US conference in 2020.
+The goal will be to make closer to folks on the east coast,
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
We want to make this one somewhere closer to folks on the east coast,
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+but we haven't figured out where to host it yet.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
but we haven't figured out exactly where to host it yet.
```

File: docs/blog/write-the-docs-mid-year-update-2019.rst [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+Currently we're in the process of trying to find suitable venues.
+We have more details in our :doc:`request for venues </conf/cfp/east-2020>`.
+If you know of a good venue that you think would work well for a Write the Docs conference,
+please let us know!
+
+Meetups
+-------
+
+We're almost at 50 meetups now, including a newly created `remote meetup <http://writethedocsremote.tk/#events/>`_!
+Kick off event 09/10/2019 8:30pm EDT.
```
Comment (plaindocs) [inline docs/blog/write-the-docs-mid-year-update-2019.rst]:
```suggestion
The kick-off event was a few days ago, and we're eager to hear how it went.
```

**Review by plaindocs — COMMENTED** [review]
Mostly suggestions to improve flow.

File: docs/conf/cfp/east-2020.rst [hunk 9]
```diff
@@ -0,0 +1,40 @@
+Write the Docs East 2020 Call for Venues
+========================================
+
+We're also excited to start planning another US conference in 2020.
+The goal will be to make closer to folks on the east coast,
+but we haven't figured out exactly where to host it yet.
+
+The plan is to host it sometime in the fall,
```
Comment (plaindocs) [inline docs/conf/cfp/east-2020.rst]:
```suggestion
We're thinking of hosting it sometime in the fall,
```

File: docs/conf/cfp/east-2020.rst [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+so that it is nicely spaced out with our Portland conference.
```
Comment (plaindocs) [inline docs/conf/cfp/east-2020.rst]:
```suggestion
to give folks who want to attend both a chance to, and so things don't get too hectic for the organizing teams. 
```

File: docs/conf/cfp/east-2020.rst [hunk 11]
```diff
@@ -0,0 +1,40 @@
+Write the Docs East 2020 Call for Venues
+========================================
+
+We're also excited to start planning another US conference in 2020.
+The goal will be to make closer to folks on the east coast,
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: Perhaps "The goal is to make it closer to..." instead? I think "it" is missing after "make"

File: docs/conf/cfp/east-2020.rst [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+but we haven't figured out exactly where to host it yet.
+
+The plan is to host it sometime in the fall,
+so that it is nicely spaced out with our Portland conference.
+
+Requirements
+------------
+
+We have some lose requirements for venues that we're looking for.
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: change "lose" to "loose" -- I also think you can eliminate the rest of the sentence after "venues"

We have some loose requirements for venues.

File: docs/conf/cfp/east-2020.rst [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+It's OK if a venue doesn't fit one or two of these,
+but they are the general goal of what we're looking for.
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: Maybe "...but this is a general picture of what we're looking for." instead?

File: docs/conf/cfp/east-2020.rst [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+
+* Fits up to 250 people
+    - Preferably with some tables along with chairs, but not a requirement
+* Has character. We are quite unlikely to try and use a conference center or hotel basement, as that is really not the vibe we're going for.
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: Maybe change the second sentence to:

We are unlikely to use a conference center or hotel space--it's not the vibe we're going for.

File: docs/conf/cfp/east-2020.rst [hunk 15]
```diff
@@ -0,0 +1,40 @@
...
+* Has two or three additional rooms for an Unconference room (large or multiple small/medium), Staff/Green room (small), and Quiet room (small). The more additional rooms the better
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: period after "better"

File: docs/conf/cfp/east-2020.rst [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+so that it is nicely spaced out with our Portland conference.
+
+Requirements
+------------
+
+We have some lose requirements for venues that we're looking for.
+It's OK if a venue doesn't fit one or two of these,
+but they are the general goal of what we're looking for.
+
+* Fits up to 250 people
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: period after "people"

got to have a couple nitpicks :D

File: docs/conf/cfp/east-2020.rst [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+    - Preferably with some tables along with chairs, but not a requirement
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: period after "requirement"

File: docs/conf/cfp/east-2020.rst [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+* Has two or three additional rooms for an Unconference room (large or multiple small/medium), Staff/Green room (small), and Quiet room (small). The more additional rooms the better
+* Not too expensive. Around or below $10,000 for the rental is a good starting point, but even better if it's free with a catering minimum or other deal.
+
+Questions
+---------
+
+Here are some good questions to answer when you think about the venue:
+
+* Is the venue easily reachable by public transit and/or does it have sufficient parking, depending on the transit options in your local area?
+* Is there affordable accommodations nearby?
```
Comment (hillaryfraley) [inline docs/conf/cfp/east-2020.rst]: Change "Is" to "Are"

Are there affordable accommodations nearby?

**Comment by ericholscher** [comment]
Alright, this is now mostly ready in my eyes. I updated the ship date to tomorrow if we can get it reviewed by then.

**Pull request merged**